index.html

```diff
@@ -261,18 +261,24 @@ <h3 class="section-title">Find Us</h3>
     </div>
     <div class="container">
       <div class="row">
-        <div class="affiliates col-md-6 to-animate">
+        <div class="affiliates col-md-4 to-animate">
           <h3 class="section-title">Proud Supporter of:</h3>
           <a href="https://opencollective.com/dltxio" target="_blank">
             <img src="https://opencollective.com/static/images/opencollectivelogo-footer-n.svg" width=300 />
           </a>
         </div>
-        <div class="affiliates col-md-6 to-animate">
+        <div class="affiliates col-md-4 to-animate">
           <h3 class="section-title">Proud Member of:</h3>
           <a href="https://blockchainaustralia.org/member/dltx/" target="_blank">
             <img src="images/Blockchain-Australia.png" width=300 />
           </a>
         </div>
+        <div class="affiliates col-md-4 to-animate">
+          <h3 class="section-title">Proud Sponsor of:</h3>
+          <a href="https://blockchainaustralia.org/member/dltx/" target="_blank">
```

Comment:
Sonarlint wants a `rel="noopener"` on that link

Reply:
`rel="noreferrer noopener"` pls

Reply:
done